tests/buyers/views/test_buyers.py

```diff
@@ -315,7 +392,7 @@ def test_copy_brief_and_redirect_to_copied_brief_edit_title_page(self):
     def test_copy_brief_for_expired_framework_redirects_to_edit_page_for_new_framework(self):
         self.data_api_client.get_brief.return_value = api_stubs.brief()  # dos1 brief
 
-        new_brief = self.brief  # dos2 brief
+        new_brief = self.brief.copy()  # dos2 brief
```

Comment:
Yeah i've stuck copy on the other instances.
Don't think I explained very well what I meant above. 
The `self.brief` object is mutable so without copy we end up mutating one object that is both our `brief` and our `new_brief`
```
self.brief = api_stubs.brief()
new_brief = self.brief
new_brief["briefs"]["id"] = 1235
new_brief['briefs']['id'] == self.brief['briefs']['id']
>>> True
```